proposals/wgs/wg-governance.md

```diff
@@ -0,0 +1,219 @@
+be eligible to hold a leadership role within a WG.
+- WG *MAY* prefer various levels of domain knowledge depending on the
+role. This should be documented.  
+- People management interests - there's a lot of us!
+
+#### Escalations
+
+- Lead membership disagreements *MAY* be escalated to the WG Chairs. WG Chair
+membership disagreements may be escalated to the Steering Committee.
+
+#### On-boarding and Off-boarding Leads
+
+- Leads *MAY* decide to step down at anytime and propose a replacement.  Use
+lazy consensus amongst other Leads with fallback on majority vote to accept
+proposal.  The candidate *SHOULD* be supported by a majority of WG contributors
+ or the Subproject contributors (as applicable).
+- Leads *MAY* select additional leads through a [super-majority] vote
+amongst leads. This *SHOULD* be supported by a majority of WG contributors or
+Subproject contributors (as applicable).
+
+### Chair
+
+- Number: 2-3
+- Membership tracked in [wg.yaml]  
+  - If no tech lead role is present, Chair assumes responsibilities from [#tech-lead] section.
+  
+  In addition, run operations and processes governing the WG:
+
+- *SHOULD* define how priorities and commitments are managed and delegate to other leads as needed
+- *SHOULD* drive charter changes (including creation) to get community buy-in but *MAY* delegate content creation to WG contributors
+- *SHOULD* identify, track, and maintain the WGs enhancements for current
+  release and serve as point of contact for the release team, but *MAY* delegate
+   to another Lead to fulfill these responsibilities
+  - *MAY* delegate the creation of a WG roadmap to other Leads
+  - *MUST* organize a main group meeting and make sure [wg.yaml] is up to date
+  including subprojects and their meeting information but *SHOULD* delegate the
+  need for subproject meetings to subproject owners  
+  - *SHOULD* facilitate meetings but *MAY* delegate to other Leads or future
+  chairs/chairs in training
+  - *MUST* ensure there is a maintained CONTRIBUTING.md document in the
+  appropriate WG folder if the contributor experience or on-boarding knowledge
+  is different than in the general [contributor guide]. *MAY* delegate to
+  contributors to create or update.
+  - *MUST* ensure meetings are recorded and made available
+  - *MUST* report activity with the community via kubeflow-discuss mailing list at least
+  once a quarter (slides, video from kubecon, etc)
+  - *MUST* coordinate sponsored SIG updates to the WG and the wider
+  community  
+- *MUST* coordinate communication and be a connector with other community
+ groups like WGs and the Steering Committee but *MAY* delegate the actual
+ communication and creation of content to other contributors where
+ appropriate  
+- *MUST* provide quarterly updates through our community channels: twice a year
+to Kubeflow-discuss@googlegroups.com mailing list and twice a year presenting at
+the weekly community meeting  
+
+### Tech Lead
```

Comment:
This is for entire Kubeflow project or different ML areas like training, orchestration, serving, etc? 
Most of the areas only have one subproject now.

Reply:
Is this equivalent to sig lead or co-chair in Kubernetes community?

Reply:
@Jeffwan  "Kubeflow SG" is a typo should be Kubeflow WG; where did you see it?

>  twice a year presenting at the weekly community meeting  

These are reporting guidelines for each WG. The idea is to set expectations around each WG communicating what its doing to the broader community.

Kubeflow WGs = Kubernetes Sigs.

So  KF tech leads and chairs are equivalent to K8s SIg TLs and CHairs

Reply:
Here. https://github.com/kubeflow/community/pull/348#issue-438760778

Reply:
Sorry should be sig